x-pack/plugins/security_solution/public/detection_engine/rule_exceptions/components/exception_item_card/meta.tsx

```diff
@@ -133,9 +133,9 @@ export const ExceptionItemCardMetaInfo = memo<ExceptionItemCardMetaInfoProps>(
                     <EuiToolTip content={listAndReferences.name} anchorClassName="eui-textTruncate">
                       <SecuritySolutionLinkAnchor
                         data-test-subj="ruleName"
-                        deepLinkId={SecurityPageName.rules}
+                        deepLinkId={SecurityPageName.exceptions}
                         // TODO: Update to list details URL once available
-                        path={getRuleDetailsTabUrl(listAndReferences.id, RuleDetailTabs.alerts)}
+                        path={`/details/${listAndReferences?.id}`}
```

Comment:
I believe the link won't work bec, the route works with the `list_id`, not the `id` 🤔
{`/details/${listAndReferences?.list_id}`}